tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
```

Comment:
Just a note that I should come back and style package names consistently later. We decided in another doc that "Package tracing provides" and "Use package tracing for" are both valid (i.e., that we don't need monospacing or lowercase).

Instead of making this change everywhere now, I'll copyedit the whole set with all the context closer to launch. Just something to think about.